### DISCUSSION TITLE
Form Dialog Tutorial

### DISCUSSION THREAD

**Discussion by aparnaji** [microsoft/BotFramework-Composer#6411] [General]
Is there any Form Dialog Tutorial on how we can use Form Dialog in Composer, Create Schema files or Schema upload (.form files) is there a way we can create schema files from existing PDF's forms

**Comment by hatpick** [comment]
You can find a tutorial here:
https://github.com/microsoft/BotFramework-Composer/blob/main/docs/preview%20features/form-dialogs.md

currently extracting forms from PDF is not supported.
